pkg/okctlapplication/kaexintegration.go

```diff
@@ -0,0 +1,83 @@
+package okctlapplication
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/oslokommune/kaex/pkg/api"
+)
+
+/*
+AcquireApplication returns an okctl Application based on stdin or a file
+*/
+func AcquireApplication(path string) (api.Application, error) {
+	var (
+		rawApplication []byte
+		err            error
+	)
+
+	if path == "-" {
+		rawApplication, err = ioutil.ReadAll(os.Stdin)
```

Comment:
We should pass in okctl's `In` here, instead of `os.Stdin`, https://github.com/oslokommune/okctl/blob/master/pkg/context/context.go#L28. This makes it easier to test. We could also use the `io.Reader` interface for both of these branches in the if statement, e.g.:

```go
r = os.Open(...) // returns an io.Reader
```

Then we could read in the bytes from `r`, which can be either an opened file or `os.Stdin`, or whatever else implements the `io.Reader` interface.